### ISSUE TITLE
Install Button

### ISSUE THREAD

**Issue by MWE001** [vendethiel/ezarena#83]
Not sure if this is a bug or a known issue already but I can not get the install button to work in Chrome. If I use Firefox it works just fine other than some errors due to it being a WAMP stack. I uploaded to a production server at a data center and the WAMP errors went away but the install button on chrome still refused to work.

**Comment by vendethiel** [comment]
really? that's weird.

I just did it with my chrome on mac:
![image](https://user-images.githubusercontent.com/199499/42579245-29bc7b7a-8528-11e8-862a-53398cb91be9.png)

it seems to work a-okay

**Comment by MWE001** [comment]
Hmm that is weird. Let me do some more testing and I'll report back. I have tried a few different times and IE works, FireFox works but Chrome does not. I click the install button and it literally does nothing. It is as if the button is not even a button but a mere image only. 

I will get out my wife's laptop and test on it as well. Maybe my Chrome is being stupid again. On occasion I have to uninstall it and reinstall it. Not sure why.

**Comment by vendethiel** [comment]
could it just be that it's very slow? I'm not sure what could be at work.

**Comment by MWE001** [comment]
I have clicked it and walked off for coffee and came back to it still there. 

When I mouse over it, I don't even get a hint that is a link (button). It does not blue outline or anything like it does in IE (Edge) or FF.

Maybe I can get Cam Studio going and get a screen capture for you if all else fails for me.

**Comment by vendethiel** [comment]
This would be interesting. I need to try on a non-mac computer I suppose. Maybe the chrome on windows has a different behavior? Weird.

**Comment by MWE001** [comment]
Still no good with Chrome on a live production server on 3 computers. Hmm.  

On a side note, is this for php7 only? I just tried on the following and it failed with the error below.


Apache Version | 2.2.31
-- | --
PHP Version | 5.6.35
MySQL Version | 5.5.51-38.2

An error occurred during installation
An error occurred while attempting to update the database

**Comment by vendethiel** [comment]
No, it's not for PHP7, it doesn't work on PHP7. This is one of the reasons I want to port the AD&R part to Integramod, really: I don't want to port yet another thing to PHP7...

**Comment by vendethiel** [comment]
The PCP mod I think I have for AD&R is actually for ADR 0.3.4 I think. I need to take a look, and be able to port it to newer versions.

It'll be fine anyway :). After all I started from Shadowtek's premod, and I got it up from 0.3.4 to 0.4.5!
